runtime/port/sysvipc/j9shmem.c

```diff
@@ -1346,15 +1360,24 @@ j9shmem_getDir(struct J9PortLibrary* portLibrary, const char* ctrlDirName, uint3
 		Assert_PRT_true(rc < 0);
 	} else {
 		if (appendBaseDir) {
-			if (omrstr_printf(buffer, bufLength, SHM_STRING_ENDS_WITH_CHAR(rootDir, '/') ? "%s%s" : "%s/%s", rootDir, J9SH_BASEDIR) >= bufLength) {
+			if (omrstr_printf(buffer,
+								bufLength,
+								SHM_STRING_ENDS_WITH_CHAR(rootDir, DIR_SEPARATOR)
+									? "%s%s"
+									: "%s" DIR_SEPARATOR_STR "%s",
+								rootDir,
+								J9SH_BASEDIR) >= bufLength
+			) {
```

Comment:
Excessive indenttion (compare with lines 1335-1341):
```c
			if (omrstr_printf(
					buffer,
					bufLength,
					SHM_STRING_ENDS_WITH_CHAR(rootDir, DIR_SEPARATOR)
						? "%s%s" DIR_SEPARATOR_STR
						: "%s" DIR_SEPARATOR_STR "%s" DIR_SEPARATOR_STR,
					rootDir,
					J9SH_BASEDIR) >= bufLength
			) {
```
Notice the formats always end with `DIR_SEPARATOR_STR`.